packages/larva-patterns/modules/hero-image-story/hero-image-story.prototype.js

```diff
@@ -0,0 +1,41 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const o_figure = clonedeep( require( '../../objects/o-figure/o-figure.prototype.js' ) );
+const story = clonedeep( require( '../story/story.center-vertical' ) );
+const article_kicker_prototype = clonedeep( require( '../article-kicker/article-kicker.prototype' ) );
+
+o_figure.c_figcaption.c_figcaption_caption_markup = 'test';
+o_figure.c_lazy_image.c_lazy_image_placeholder_url = 'https://picsum.photos/1000/1000';
+o_figure.c_lazy_image.c_lazy_image_classes = 'lrv-a-wrapper-100vw@mobile-max';
+o_figure.c_lazy_image.c_lazy_image_crop_class = 'lrv-a-crop-3x2';
+o_figure.o_figure_classes = 'lrv-a-glue-parent@tablet';
+o_figure.c_figcaption.c_figcaption_classes = 'lrv-u-color-white lrv-a-glue@tablet lrv-a-glue-parent@mobile-max lrv-u-flex lrv-u-flex-direction-column lrv-a-glue--b-0 lrv-a-glue--l-0 lrv-u-width-20p lrv-u-width-100p@mobile-max lrv-u-align-items-center';
+o_figure.c_figcaption.c_figcaption_credit_classes = 'lrv-a-font-primary-xl lrv-u-padding-tb-125@mobile-max lrv-u-border-b-1@mobile-max lrv-u-text-align-center@tablet lrv-u-text-transform-uppercase lrv-u-font-family-secondary';
+o_figure.c_figcaption.c_figcaption_credit_text = 'Prince Dead At 57';
+o_figure.c_figcaption.c_figcaption_caption_classes = 'u-background-white@mobile-max lrv-a-glue@mobile-max lrv-a-glue--b-unset@mobile-max lrv-a-glue--t-0 lrv-a-glue--l-0 lrv-u-transform-translateY-n50p lrv-u-font-family-secondary lrv-u-font-size-20 u-line-height-18 u-letter-spacing-012 lrv-u-border-t-5 lrv-u-border-b-1 lrv-u-padding-tb-050 lrv-u-padding-lr-125';
+o_figure.c_figcaption.c_figcaption_caption_markup = '1958 – 2021';
+
+//story.c_lazy_image = false;
+story.c_dek = false;
+story.c_timestamp = false;
+story.c_link_bottom = false;
+story.c_span = false;
+story.article_kicker = clonedeep( article_kicker_prototype );
```

Comment:
not a blocker but would be good if we can add these clone stmts at the top all together